tc/sgx/trusted_worker_manager/enclave/signup_enclave.cpp

```diff
@@ -45,6 +45,8 @@
 #include "signup_enclave.h"
 #include "verify-report.h"
 
+std::string g_Kme_Verifying_key = "";
```

Comment:
This variable is not required in Avalon 0.6. Can be removed.

Reply:
Sure.